input/software_config.json

```diff
@@ -11,7 +11,8 @@
         {"name": "roce_plugin"},
         {"name": "jupyter"},
         {"name": "pytorch"},
-        {"name": "tensorflow"}
+        {"name": "tensorflow"},
+        {"name": "gaudi", "version": "1.16.2-2"}
```

Comment:
subgroup for habana similar to amdgpu and rocm. If guadi present in softwares then only habana stack should be setup